command.php

```diff
@@ -337,7 +339,7 @@ private function tearDownThemes() {
 	 *
 	 * @return bool
 	 */
-	public function is_woocommerce_blocks_active(): bool {
+	public function isWoocommerceBlocksActive(): bool {
```

Comment:
What are your thoughts on making this function `private`:

```suggestion
	private function isWoocommerceBlocksActive(): bool {
```

As it's currently `public`, the function appears in the help menu:

<img width="764" alt="Screenshot 2022-07-08 at 12 08 57" src="https://user-images.githubusercontent.com/3323310/177971363-cda466b9-4aa4-4860-9fe3-991435cbf3ce.png">

I have the feeling that it's rather unlikely that someone calls `wp woo-test-environment isWoocommerceBlocksActive` directly.

Reply:
Oops, makes total sense. Didn't know that public functions were showing as subcommands! 😅